src/extension/executors/notebookTerminal.ts

```diff
@@ -0,0 +1,24 @@
+import { NotebookCell, NotebookCellOutput, NotebookCellOutputItem, window } from 'vscode'
+
+import { Kernel } from '../kernel'
+import { replaceOutput } from '../utils'
+import { OutputType } from '../../constants'
+import NotebookTerminal from '../notebookTerminal'
+
+export async function NotebookTerminalExecutor(cell: NotebookCell, kernel: Kernel) {
```

Comment:
This doesn't really fit in line with the rest of the "executors."

I think it would make more sense to have this as a method on the `NotebookTerminal` class - something like `openNotebookTerminal`. I feel like that would be more transparent as to what this code actually does.